packages/main/src/ComboBox.ts

```diff
@@ -971,6 +964,22 @@ class ComboBox extends UI5Element implements IFormInputElement {
 		}
 
 		if ((isTabNext(e) || isTabPrevious(e)) && this.open) {
+			if (this._isValueStateFocused && this.valueStateMessageText.length) {
+				const hasLinkInsideValueStateMessages = this.valueStateMessage.find(item => item.querySelectorAll("a").length > 0);
+				if (hasLinkInsideValueStateMessages) {
+					e.stopImmediatePropagation();
+					e.preventDefault();
+					this._isValueStateFocused = false;
+					// Focus the first link inside the value state message if visual focus is on the message and the user presses tab
+					setTimeout(() => {
+						const firstLink = this._getPicker().querySelector("*[slot='header']")!.querySelector("*[slot='valueStateMessage']")!.querySelector("a")!;
+						firstLink.tabIndex = 0;
+						firstLink.focus();
+					}, 100);
+					return;
+				}
+			}
+
```

Comment:
Could we please think of another approach for this use case as this seems to be very complex and unreliable. If you think that handling this case means spending a lot of time on research we may create another BLI for this case only ( I have discussed this with our PO ).